docs/pages/blog/toolpad-use-cases.md

```diff
@@ -0,0 +1,110 @@
+
+<video controls width="100%" height="auto" style="contain" alt="zendesk first reply in devmode">
+  <source src="/static/blog/toolpad-use-cases/zendesk-first-reply-dev.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+We opted for Toolpad since Metabase doesn't support importing data from REST APIs.
+This is possible in Google Sheets but it requires writing a lot of JS code, and since we wanted to embed it in a [Notion page](https://mui-org.notion.site/KPIs-1ce9658b85ce4628a2a2ed2ae74ff69c?pvs=4#3974cb6ed12b4c5a9013bac63113e3bc), Toolpad was the ideal choice.
+Toolpad handles state management and routing, and simplifies query building and data binding, removing the need to write glue code.
+
+You can explore both of the aforementioned apps in dev mode on your device by running the underlying [Node application](https://github.com/mui/mui-public/tree/HEAD/tools-public).
+The next two apps discussed are internal to MUI.
+
+## 3. Manage overdue invoices
+
+We have an internal operations process to remind customers about overdue invoices: reminder emails are sent out at regular intervals following the due date.
+In the Toolpad app, the operations team can view a table of all customers who need to be contacted.
+They select one, email them from Zendesk, and update the status to `CONTACTED`, which is then written back to the database.
+This is a private app for internal use that can't be shared, but the following video gives a quick demo:
+
+<video controls width="100%" height="auto" style="contain" alt="overdue invoice page">
+  <source src="/static/blog/toolpad-use-cases/overdue-invoice.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+Before this, we relied on a combination of Metabase and Google Sheets to fetch and edit records—but this was cumbersome to operate and sometimes caused data inconsistency issues.
+Thanks to Toolpad we've managed to bring it all under one roof, dramatically improving our operations team's efficiency and productivity.
+
+## 4. Contributor payout
+
+We have a script to fetch monthly payout data for contributors to the MUI Store.
+Our operations team is responsible for paying contributors, but the script proved too technically challenging for them to run without help from our engineers.
+We solved this problem by importing the script into Toolpad and creating a UI for it.
+The video below shows how a user can select the dates, run the script, and receive text that's properly formatted to copy and paste directly into Slack communications:
+
+<video controls width="100%" height="auto" style="contain" alt="contributor payout page">
+  <source src="/static/blog/toolpad-use-cases/contributor-payout.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+With the release of [authentication](https://mui.com/toolpad/concepts/authentication/), we will now enable Google login for the above apps.
```

Comment:
Will we update these apps with authentication and then update this blog?

Reply:
We may not update this app in the near future as it is blocked due to https://github.com/mui/mui-toolpad/issues/3223. I just wanted to plug the authentication feature here. Do you suggest I should remove this line?